_includes/live-demos/editor-dfree/style.css

```diff
@@ -8,14 +8,14 @@
   padding: 20px;
 }
 .demo-dfree *[contentEditable="true"]:hover {
-  outline: 1px solid #2276d2;
+  outline: 1px solid {{site.primaryColor}};
```

Comment:
This is incorrect, as you don't want the hover style to be the same as the content, otherwise you won't see it.

Reply:
There is no border on the content, the border only appears on hover to indicate that you are hovering over an editor instance. Blue on white.

Reply:
Oh right, this is an outline sorry :facepalm:

Reply:
Worth noting the website uses `#145CA4` for hover color normally, but that's a different case to using it for `outline`.